### PR TITLE
First Capstone Project

### PR DESCRIPTION
**Pull request by Christabelle143**
# In this project, I;
- Created a new repo for the capstone project which  in my case is a website for a restaurant
- Created an home.html, about.html, style.css and index.js.
- Created the side using HTML, CSS, and JavaScript codes.
###  I implemented the following according to the activity given;
- Created the home and about page for the website.
- Made the home and about page have versions for two different screens that's mobile and desktop screens.
- The home page having a link in the menu to the about page.
- The logo in the header links to the home page.
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu.
- Made the website responsive using Bootstrap classes
- Downloaded images to make the design attractive.
- Created the section "Some dishes" dynamically in JavaScript.
- Add linters file
- Finally added a descriptive README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) from who I got my inspiration.
+- A great thanks to My coding partners, morning session team, and standup team Partners for their contributions.
+- Thanks to Code Reviewers for the insightful feedback.
+
+## 📝 License
+
```
Comment (MahmoudBakr23) [inline README.md:62]: - [ ] Please fix the MIT link for a completely professional README file

File: home.html [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+        <li>
+          <a href="#">Program</a>
+        </li>
+        <li>
+          <a href="#">Join</a>
+        </li>
+        <li class="border-line">
+          <a href="#"> Day Menu</a>
+        </li>
+      </ul>
```
Comment (MahmoudBakr23) [inline home.html]: - [ ] Well done creating the navbar in the Desktop version and smaller screens. However, in the mobile screen version view, the design breaks! Please fix that to match the given design
Screenshot (2)
t

File: home.html [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    </nav>
+    <section class="intro">
+      <div class="intro-content">
+        <h2 class="intro-message">"Your Satisfaction Our Pride"</h2>
+        <h1>Welcome to Chef Food Restaurant</h1>
+      </div>
+      <div class="intro-desc">
+        <p>
```
Comment (MahmoudBakr23) [inline home.html:111]: - [ ] In the mobile screen under nearly 450-width view, the design breaks. Please fix it to match the given design.
Screenshot (1)

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Christabelle143 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Correct Linters
- Descriptive README, PR
- Nice presentation
- Well done following the design in the desktop screen version

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [ ] Please apply the design changes in both Home and About pages

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: home.html (hunk 2 above)
Comment (Christabelle143) [reply home.html]: There is no given design for the menu option appearing over the page. That's how i implemented it. So when you say it breaks i don't understand??. Did u mean I should give it some margin right??

File: home.html (hunk 3 above)
Comment (Christabelle143) [reply home.html:111]: Hello @MahmoudBakr23 
    Just looked back at my design and i think it's ok. But will be helpful if you elaborate more on your point of my mobile design breaking under nearly 450-width view. Just checked it and it was very ok. It starts breaking at 350 which i think is quite a very small width. The image couldn't be responsive to that level. As it has it minimum screen.

Comment (Christabelle143) [reply home.html:111]: Hello @MahmoudBakr23

Comment (Christabelle143) [reply home.html:111]: Hey  @MahmoudBakr23 
I can't find my design breaking around 450 width. It's working fine till 360 width.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Christabelle143, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**